src/probability/independence.lean

```diff
@@ -309,6 +308,119 @@ begin
   exact indep_sets.indep_aux h2 hp2 hpm2 hyp ht ht2,
 end
 
+variables {α ι : Type*} {m0 : measurable_space α} {μ : measure α}
+
+lemma pi_system_indep_insert {π : ι → set (set α)} {a : ι} {S : finset ι}
```

Comment:
The name of this lemma does not really match the symbols in the statement of the lemma.